src/coredump/coredump-ratelimit.h

```diff
@@ -0,0 +1,9 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include <inttypes.h>
+#include <sys/types.h>
```

Comment:
Please sort

Reply:
Addressed the comment. Please check once.

Reply:
nit: sys/stat.h should be above sys/types.h